velox/exec/fuzzer/WindowFuzzer.cpp

```diff
@@ -96,20 +235,42 @@ std::tuple<std::string, bool> WindowFuzzer::generateFrameClause() {
     maxKValue = kMax;
   }
 
-  auto frameBound =
-      [minKValue, maxKValue, this](
-          core::WindowNode::BoundType boundType) -> const std::string {
+  auto frameBound = [&](core::WindowNode::BoundType boundType,
+                        std::string columnName,
+                        bool isStartBound) -> const std::string {
     // Generating only constant bounded k PRECEDING/FOLLOWING frames for now.
     auto kValue = boost::random::uniform_int_distribution<int64_t>(
         minKValue, maxKValue)(rng_);
     switch (boundType) {
       case core::WindowNode::BoundType::kUnboundedPreceding:
         return "UNBOUNDED PRECEDING";
       case core::WindowNode::BoundType::kPreceding:
+        if (windowType == core::WindowNode::WindowType::kRange) {
+          if (windowType == core::WindowNode::WindowType::kRange) {
+            if (isPrestoQuery) {
+              return isStartBound
+                  ? fmt::format("{} PRECEDING", prestoFrames_[0].first)
+                  : fmt::format("{} PRECEDING", prestoFrames_[0].second);
```

Comment:
Pass prestoFrames_[0].first or prestoFrames_[0].second to frameBound() so that this branch on isStartBound is not needed.

Reply:
Updated, thanks.